src/components/Visualizations/GraphPanes/YearsGraphPane.tsx

```diff
@@ -93,10 +96,27 @@ export const YearsGraphPane = ({ data, onApplyCondition }: IYearsGraphPaneProps)
     }
   }, [baseGraph, debouncedRange]);
 
+  const getCSVDataContent = () => {
+    let content = 'Year, Article Count, Ref Count';
+
+    baseGraph.data.forEach(({ year, refereed, notrefereed }) => {
+      content += `\n${year},${refereed + notrefereed},${refereed}`;
+    });
+
+    return content;
+  };
```

Comment:
```suggestion
  const getCSVDataContent = into(
    'Year, Article Count, Ref Count',
    map(({ year, refereed, notrefereed }) => `\n${year},${refereed + notrefereed},${refereed}`))
  );
```

Haven't actually tested this out, but you should be able to do something like this if you wanted to use the utils.  (will need to also add `into` and `map` from ramda)